monkey/monkey_island/cc/models/monkey_ttl.py

```diff
@@ -38,3 +38,16 @@ def create_ttl_expire_in(expiry_in_seconds):
     }
 
     expire_at = DateTimeField()
+
+
+def create_monkey_ttl_document(expiry_duration_in_seconds):
+    """
+    Create a new Monkey TTL document and save it as a document.
+    :param expiry_duration_in_seconds:  How long should the TTL last for. THIS IS A LOWER BOUND - depends on mongodb
+    performance.
+    :return: The TTL document. To get its ID use `.id`.
+    """
+    # The TTL data uses the new `models` module which depends on mongoengine.
+    current_ttl = MonkeyTtl.create_ttl_expire_in(expiry_duration_in_seconds)
```

Comment:
What can we do with the document that we get from `create_ttl_expire_in(expiry_duration_in_seconds)` besides saving?

Reply:
This for example: 

![image](https://user-images.githubusercontent.com/48879847/61641715-4b573380-aca8-11e9-808c-930536452b22.png)